packages/libs/snap/README.md

```diff
@@ -12,10 +9,665 @@ many more.
+  params: {
+    snapId: defaultSnapOrigin,
+    request: {
+      method: 'kda_storeNetwork',
+      params: {
+        network: {
+          name: 'New Network',
+          chainId: 1,
+          networkId: 'new-network-id',
+          nodeUrl: 'https://new-network-node.url',
+          blockExplorerTransaction: 'https://explorer.url/tx/{txId}',
+          blockExplorerAddress: 'https://explorer.url/address/{address}',
+          isTestnet: true,
+        },
+      },
+    },
+  },
+});
+```
+
+**Response**: [`SnapNetwork`](#type-definitions)
+
+---
+
+### kda_deleteNetwork
+
+**Description**: Deletes a network from the Kadena Snap.
+
+**Request Example**:
+
+```javascript
+await window.ethereum.request({
+  method: 'wallet_invokeSnap',
+  params: {
+    snapId: defaultSnapOrigin,
+    request: {
+      method: 'kda_deleteNetwork',
+      params: { networkId: '<networkId>' },
+    },
+  },
+});
+```
+
+**Response**: None
+
+---
+
+### kda_getActiveNetwork
+
+**Description**: Retrieves the active network from the Kadena Snap.
+
+**Request Example**:
+
+```javascript
+const activeNetwork = await window.ethereum.request({
+  method: 'wallet_invokeSnap',
+  params: {
+    snapId: defaultSnapOrigin,
+    request: { method: 'kda_getActiveNetwork' },
+  },
+});
+```
+
+**Response**: [`GetActiveNetworkResponse`](#type-definitions)
+
+---
+
+### kda_setActiveNetwork
+
+**Description**: Sets the active network in the Kadena Snap.
+
+**Request Example**:
+
+```javascript
+await window.ethereum.request({
+  method: 'wallet_invokeSnap',
+  params: {
+    snapId: defaultSnapOrigin,
+    request: {
+      method: 'kda_setActiveNetwork',
+      params: { networkId: '<networkId>' },
+    },
+  },
+});
+```
+
+**Response**: None
+
+---
+
+### kda_setAccountName
+
+**Description**: Updates the name (or alias) of an account in the Kadena Snap.
+
+**Request Example**:
+
+```javascript
+await window.ethereum.request({
+  method: 'wallet_invokeSnap',
+  params: {
+    snapId: defaultSnapOrigin,
+    request: {
+      method: 'kda_setAccountName',
+      params: {
+        id: '<id>',
+        name: 'New Account Name',
+      },
+    },
+  },
+});
+```
+
+**Response**: None
+
+---
+
+### kda_setHardwareAccountName
+
+**Description**: Updates the name (or alias) of a hardware account in the Kadena Snap. 
+
+**Request Example**:
+
+```javascript
+await window.ethereum.request({
+  method: 'wallet_invokeSnap',
+  params: {
+    snapId: defaultSnapOrigin,
+    request: {
+      method: 'kda_setHardwareAccountName',
+      params: {
+        id: '<id>',
+        name: 'New Hardware Account Name',
+      },
+    },
+  },
+});
+```
+
+**Response**: None
+
+---
+
+### kda_signTransaction
+
+**Description**: Signs a transaction using the Kadena Snap. The transaction is expected to be a stringified JSON of the type [`ICommandPayload`](https://github.com/kadena-community/kadena.js/blob/f88301538ee41b77b957edf58a8a5edb4fba44cb/packages/libs/types/src/PactCommand.ts#L176).
```

Comment:
I wasn't sure where to fine a more permanent documentation link for `ICommandPayload`.

I derived that the expected request type was this from the following areas of the code:
- https://github.com/kadena-community/kadena.js/blob/f88301538ee41b77b957edf58a8a5edb4fba44cb/packages/libs/snap/src/services/signTransaction.ts#L53
- https://github.com/kadena-community/kadena.js/blob/f88301538ee41b77b957edf58a8a5edb4fba44cb/packages/libs/snap/src/services/signTransaction.ts#L75
- https://github.com/kadena-community/kadena.js/blob/f88301538ee41b77b957edf58a8a5edb4fba44cb/packages/libs/snap/src/services/signTransaction.ts#L22

But would appreciate it if someone else could verify as well.

Reply:
as we discussed, IPactCommand is more current